pucoti.py

```diff
@@ -382,6 +384,20 @@ def table(
         return surf
 
 
+def adjust_window_size(window, scale_factor: float):
+    new_width = window.size[0] * scale_factor
+    new_height = new_width/2.75
```

Comment:
Is there a specific reason for the 2.75?

Reply:
oh yeah, it should be based on the provided dimensions not the default. Thanks :)

Reply:
Okay! Then I think it's probably simpler to use `new_height = window.size[1] * scale_factor` for simplicity